src/Symfony/Component/HttpFoundation/RequestStack.php

```diff
@@ -106,4 +106,11 @@ public function getSession(): SessionInterface
 
         throw new SessionNotFoundException();
     }
+
+    public function reset(): void
```

Comment:
should it be name `resetRequestFormats` instead ? It is not resetting the stack itself so the name `reset` is confusing.

Reply:
Better indeed, updated